src/mixins/atoms.js

```diff
@@ -94,6 +96,11 @@ export const AtomsMixin = (superclass) =>
                 });
                 sphereMesh.name = `${element}-${atomicIndex}`;
                 atomsGroup.add(sphereMesh);
+
+                const text = sphereMesh.name.split("-")[0];
+                const label = this.createLabel(text, `label-for-${sphereMesh.uuid}`);
```

Comment:
Following this method through `createSprite` to `createTextTexture`, it looks like we create distinct `Texture` objects for each atom, even if they are of the same element. I think we still need distinct `Sprite` objects but they can probably recycle `Textures` if they are of the same element.

Reply:
Yes, you are right. I need to reuse the textures.